packages/api-v2/src/auth/auth.controller.ts

```diff
@@ -11,34 +14,67 @@ import {
   LoginStudentDto,
   CreateStudentDto,
 } from "../../../common";
+import { Response } from "express";
 
 @Controller("auth")
 export class AuthController {
   constructor(private readonly authService: AuthService) {}
 
   @Post("register")
   public async register(
+    @Res({ passthrough: true }) response: Response,
```

Comment:
we need to add `passthrough` here because since we need to set cookie in response, when we add @Res, we will need to handle the response ourselves. [See here in doc](https://docs.nestjs.com/controllers#appendix-library-specific-approach)